docs/release-notes/8.0.41-32.md

```diff
@@ -0,0 +1,91 @@
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.41 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-41.html).
+
+## Improvements
+
+[PXC-4469](https://perconadev.atlassian.net/browse/PXC-4469): Implements the [Clone plugin for State Snapshot Transfer (SST) Method](https://docs.percona.com/percona-xtradb-cluster/8.0/clone-sst.html).
+
+## Bug fixes
+
+* [PXC-4173](https://perconadev.atlassian.net/browse/PXC-4173): The Percona XtraDB Cluster node stalled when parallel replication workers executed DDLs through an asynchronous node.
+
+* [PXC-4390](https://perconadev.atlassian.net/browse/PXC-4390): The `wsrep_group_commit_queue` option was removed because caused deadlock bugs.
+
+* [PXC-4498](https://perconadev.atlassian.net/browse/PXC-4498): The Percona XtraDB Cluster node stalled and caused the cluster to stop responding.
+
+* [PXC-4393](https://perconadev.atlassian.net/browse/PXC-4393): On the donor side, if the pipeline had more than two commands, only the last command’s failure was detected, but the error was shown for the second-to-last command.
+
+* [PXC-4404](https://perconadev.atlassian.net/browse/PXC-4404): Setting the `wsrep_preordered=ON` option caused protocol violations.
+
+* [PXC-4504](https://perconadev.atlassian.net/browse/PXC-4504): The `DROP PROCEDURE/FUNCTION IF EXISTS` option generated a local Global Transaction Identifier (GTID) event.
+
+* [PXC-4506](https://perconadev.atlassian.net/browse/PXC-4506): The asynchronous replication threads on a Percona XtraDB Cluster node hung when resuming from an applier error after it was fixed.
+
+* [PXC-4509](https://perconadev.atlassian.net/browse/PXC-4509): The asynchronous replication hung on the Percona XtraDB Cluster node if the replica received Global Transaction Identifier (GTID) from the source that had already been committed on it.
+
+* [PXC-4515](https://perconadev.atlassian.net/browse/PXC-4515): Multi-source replication didn't work on the Percona XtraDB Cluster node with the new asynchronous monitor logic.
+
+* [PXC-](https://perconadev.atlassian.net/browse/PXC-4516): Global Transaction Identifier (GTID) auto-positioning replication didn't work correctly on the Percona XtraDB Cluster node with the new async monitor logic.
+
+* [PXC-4573](https://perconadev.atlassian.net/browse/PXC-4573): The `mysql.pxc.sst.role` lacked the `INNODB_REDO_LOG_ARCHIVE` privilege when the State Snapshot Transfer (SST) required InnoDB redo log archiving.
+
+* [PXC-4585](https://perconadev.atlassian.net/browse/PXC-4585): If the donor node was killed during the State Snapshot Transfer (SST), the SST daemon on the joiner waited indefinitely without terminating. The clone processes continued running until manually killed, and even if the joiner server was killed, the clone daemon kept running until stopped by the user.
```

Comment:
Remove this one. It is related to clone-sst new feature.

Reply:
removed